fallout/dataset-test-nosqlbench.yaml

```diff
@@ -0,0 +1,297 @@
+                "full_name" :   "{full_name}",
+                "data" :      {document_json_without_id}
+              }
+            }
+          }
+
+  main:
+    ops:
+      # aka findOne with _id as filter
+      find-one-id:
+        space: "{space}"
+        method: POST
+        uri: <<protocol:http>>://{weighted_hosts}:<<jsonapi_port:8181>><<path_prefix:>>/v1/<<namespace:jsonapi_crud_dataset>>/<<collection:docs_collection>>
+        Accept: "application/json"
+        X-Cassandra-Request-Id: "{request_id}"
+        X-Cassandra-Token: "{token}"
+        Content-Type: "application/json"
+        ok-body: ".*\"data\".*"
+        body: >2
+          {
+            "findOne" : {
+              "filter" : {
+                "_id" : "{random_key}"
+              }
+            }
+          } 
+
+      # aka findOne with single property as filter
+      find-one-by-full-name:
+        space: "{space}"
+        method: POST
+        uri: <<protocol:http>>://{weighted_hosts}:<<jsonapi_port:8181>><<path_prefix:>>/v1/<<namespace:jsonapi_crud_dataset>>/<<collection:docs_collection>>
+        Accept: "application/json"
+        X-Cassandra-Request-Id: "{request_id}"
+        X-Cassandra-Token: "{token}"
+        Content-Type: "application/json"
+        ok-body: ".*\"data\".*"
+        body: >2
+          {
+            "findOne" : {
+              "filter" : {
+                "full_name" : "{full_name}"
+              }
+            }
+          }  
+
+      # aka find with $exist
+      find-multi-by-exists:
+        space: "{space}"
+        method: POST
+        uri: <<protocol:http>>://{weighted_hosts}:<<jsonapi_port:8181>><<path_prefix:>>/v1/<<namespace:jsonapi_crud_dataset>>/<<collection:docs_collection>>
+        Accept: "application/json"
+        X-Cassandra-Request-Id: "{request_id}"
+        X-Cassandra-Token: "{token}"
+        Content-Type: "application/json"
+        ok-body: ".*\"data\".*"
+        body: >2
+          {
+            "find" : {
+              "filter" : {
+                "{exists_key}": { "$exists": true }
+              }
+            }
+          }
+
+      # aka find with $eq and $exist and $projection
+      find-multi-by-name-with-projection:
+        space: "{space}"
+        method: POST
+        uri: <<protocol:http>>://{weighted_hosts}:<<jsonapi_port:8181>><<path_prefix:>>/v1/<<namespace:jsonapi_crud_dataset>>/<<collection:docs_collection>>
+        Accept: "application/json"
+        X-Cassandra-Request-Id: "{request_id}"
+        X-Cassandra-Token: "{token}"
+        Content-Type: "application/json"
+        ok-body: ".*\"data\".*"
+        body: >2
+          {
+            "find" : {
+              "filter" : {
+                "full_name": { "$exists": true }
+              },
+              "projection": {
+                "full_name": 1,
+                "data": 1
+              }
+            }
+          }
+
+      # aka find with filter and sort
+      find-multi-find-by-group-sort-by-sort-key:
```

Comment:
maybe names should be actually adapted to the action done?

Reply:
Will change.